packages/data-models/src/language_model_factory.js

```diff
@@ -27,12 +27,22 @@ class LanguageModelFactory {
   /**
    * Checks whether a language is supported
    *
-   * @param {string | symbol} language - Language as a language ID (symbol) or a language code (string)
+   * @param {Language | string | symbol} language - Language as a Language object,
```

Comment:
should we note that string and symbol are deprecated?

Reply:
Absolutely! I've add that note.